### ISSUE TITLE
Update subscription URL for hufilter

### ISSUE THREAD

**Issue by scripthunter7** [uBlockOrigin/uBlock-issues#3277]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

I would like to request to change the hufilter subscription URL from

https://raw.githubusercontent.com/hufilter/hufilter/master/hufilter-ublock.txt
↓↓↓↓↓
https://cdn.jsdelivr.net/gh/hufilter/hufilter@gh-pages/hufilter-ublock.txt

Reference: https://github.com/hufilter/hufilter-dev/issues/461

I tried to open a PR, but unfortunately it is not possible:

![kép](https://github.com/uBlockOrigin/uBlock-issues/assets/57285466/debde032-b1cb-410e-8f0e-fcae80bd1162)

### A specific URL where the issue occurs.

```yaml
irrelevant
```

### Steps to Reproduce

irrelevant

### Expected behavior

irrelevant

### Actual behavior

irrelevant

### uBO version

irrelevant

### Browser name and version

irrelevant

### Operating System and version

irrelevant

**Comment by garry-ut99** [comment]
>  * [x]  I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue+sort%3Aupdated-desc) to avoid opening a duplicate issue.

Duplicate of:
- https://github.com/uBlockOrigin/uBlock-issues/issues/1364

(In case you didn't miss the old thread, but purposely created a new thread: no need to do so, because new URL requests can be posted in the already existing thread, as devs still read closed threads and won't miss new comments, hence creating a new thread won't make any difference, only generates duplicates)

**Comment by scripthunter7** [comment]
> but purposely created a new thread: no need to do so

Yes, I opened a new issue on purpose. Sorry for the duplicate then, although unfortunately I can't delete it

**Comment by gorhill** [comment]
> I tried to open a PR, but unfortunately it is not possible:

If you make the change in a forked repo on your side, I will be able to pull in your commit to github/uBlock. Once this happens, you will be seen as a collaborator and from then on be able to submit pull requests directly to github/uBlock..

**Comment by scripthunter7** [comment]
@gorhill Thank you!

**Comment by scripthunter7** [comment]
@gorhill How long do you think it will take for the change to reach most users?

**Comment by garry-ut99** [comment]
> scripthunter7: How long do you think it will take for the change to reach most users?
> gorhill: `assets.json` updates every 13 days, so it should take at most that many days.

(just info) So just like last time then:
- https://github.com/uBlockOrigin/uBlock-issues/issues/1364#issuecomment-734914475
